docker-contributor/supervisor/judgedaemon2.conf

```diff
@@ -0,0 +1,6 @@
+[program:judgedaemon2]
```

Comment:
If we are updating this lets use the numprocs format: https://github.com/ajdavis/motor-blog/blob/master/etc/supervisord.conf.example#L23
Then we don’t have to copy paste the config.
Not sure if we can have only 2 out of 4 start but then we can create 2 groups